content/en/security/application_security/enabling/compatibility/nodejs.md

```diff
@@ -128,24 +125,26 @@ The following operating systems are officially supported by `dd-trace`. Any oper
 - **Threat Protection** also works at the HTTP request (input) layer, and so works for all databases by default, even those not listed in the table below.
 
 
-| Framework                 | Versions | Threat Detection supported? | Threat Protection supported? | Vulnerability Management for Code-level supported? |
-| ----------------------- | ---------- | --------------- | ---------------------------------------------- | ---------------------------------------------- |
-| [cassandra-driver][28] | `>=3`    |{{< X >}}|{{< X >}} |          |
-| [couchbase][29]        | `^2.4.2` |{{< X >}}|          |          |
-| [elasticsearch][30]    | `>=10`   |{{< X >}}|          |          |
-| [ioredis][31]          | `>=2`    |{{< X >}}|{{< X >}} |          |
-| [knex][32]             | `>=0.8`  |{{< X >}}|{{< X >}} |          |
-| [mariadb][5]           | `>=3`    |{{< X >}}|{{< X >}} |          |
-| [memcached][33]        | `>=2.2`  |{{< X >}}|{{< X >}} |          |
-| [mongodb-core][34]     | `>=2`    |{{< X >}}|{{< X >}} |          |
-| [mysql][35]            | `>=2`    |{{< X >}}|          |{{< X >}} |
-| [mysql2][36]           | `>=1`    |{{< X >}}|{{< X >}} |{{< X >}} |
-| [oracledb][37]         | `>=5`    |{{< X >}}|{{< X >}} |          |
-| [pg][38]               | `>=4`    |{{< X >}}|{{< X >}} |{{< X >}} |
-| [redis][39]            | `>=0.12` |{{< X >}}|{{< X >}} |          |
-| [sharedb][40]          | `>=1`    |{{< X >}}|{{< X >}} |          |
-| [tedious][41]          | `>=1`    |{{< X >}}|{{< X >}} |          |
-| [sequelize][42]        | `>=4`    |         |          | {{< X >}}|
+| Framework                | Versions  | Threat Detection supported? | Threat Protection supported? | Vulnerability Management for Code-level supported? |
+|--------------------------|-----------|-----------------------------|------------------------------|----------------------------------------------------|
+| [@apollo/server][43]     | `>=4`     | {{< X >}}                   | {{< X >}}                    |                                                    |
+| [apollo-server-core][44] | `>=3`     | {{< X >}}                   | {{< X >}}                    |                                                    |
```

Comment:
Maybe is not clear for the reviewers because I changed all the spacing in the tables.
apollo related lines are new.
And mongodb-core now has a ` {{< X >}}` for Code level vulnerabilities.